src/tooltip/tooltip.ts

```diff
@@ -27,7 +27,7 @@ let nextId = 0;
   selector: 'ngb-tooltip-window',
   changeDetection: ChangeDetectionStrategy.OnPush,
   host: {
-    '[class]': '"tooltip show bs-tooltip-" + placement.split("-")[0]+" bs-tooltip-" + placement',
+    '[class]': '"tooltip show bs-tooltip-" + placement.split("-")[0]+" bs-tooltip-" + placement + " " + class',
```

Comment:
We should have a check that `class` is defined and not empty, otherwise we might end up with ` undefined` in the class list. Please a test for this case.